dev-requirements.txt

```diff
@@ -3,3 +3,4 @@ pep8==1.6.1
 pyflakes==0.8.1
 mccabe==0.3
 flake8==2.3.0
+selenium==2.44.0
```

Comment:
I think it's better to have just one file plus with selenium they are just 4 packages